libs/ui/search/src/lib/record-table/record-table.component.ts

```diff
@@ -14,7 +14,7 @@ import { SortByField } from '@geonetwork-ui/common/domain/search'
   styleUrls: ['./record-table.component.css'],
 })
 export class RecordTableComponent {
-  @Input() records: CatalogRecord[] = []
+  @Input() records: any[] = []
```

Comment:
Is the `any` mandatory ?

Reply:
I know it's not ideal, but I ended up using `any` because `records` can actually have two types. It can be a user record or a catalog record (hence why I created the `User` interface you commented on). I tried to manage to have `records` to be any of these types, but couldn't. Do you have a suggestion ? I would be using `UserApiModel` in that case.

Reply:
Tried it. Then for some reason the HTML thinks ` CatalogRecord[]` is the only type and doesn't recognize the `UserApiModel` variables... Same with `CatalogRecord[] && UserApiModel[]`, where the HTML believes it's only `UserApiModel` types :frowning_face: